filecoin-proofs/src/types/sector_class.rs

```diff
@@ -10,7 +10,7 @@ pub struct SectorClass(
 impl From<SectorClass> for PoStConfig {
     fn from(x: SectorClass) -> Self {
         match x {
-            SectorClass(ss, _, ppp) => PoStConfig(ss, ppp),
+            SectorClass(ss, _, _) => PoStConfig(ss),
```

Comment:
BLOCKING: If `PoStConfig` no longer takes a proof partitions-value, then you should drop it from `SectorClass` too.

Reply:
isn't sector class used for PoRep too, and that still uses proof partitions

Reply:
Here is the `SectorClass` type:

```
pub struct SectorClass(
    pub SectorSize,
    pub PoRepProofPartitions,
    pub PoStProofPartitions,
);
```

If `PostConfig` no longer takes a proof partitions-value, then you should drop `PoStProofPartitions` from `SectorClass`.